core/pom.xml

```diff
@@ -18,12 +18,21 @@
       <module.skipComponentMetaDataProcessing>false</module.skipComponentMetaDataProcessing>
       <module.skipTestComponentMetaDataProcessing>false</module.skipTestComponentMetaDataProcessing>
    </properties>
-   <dependencies>
 
+   <dependencies>
       <dependency>
          <groupId>org.infinispan</groupId>
          <artifactId>infinispan-commons</artifactId>
       </dependency>
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-component-annotations</artifactId>
+      </dependency>
+      <!-- TODO remove dependency and only reference in annotation processor path -->
```

Comment:
Is this something to worry about still?

Reply:
Not really a worry, but it seemed slightly cleaner to define a processor path in the parent pom and put all our annotation processors explicitly there. I created [IOSPN-10284](https://issues.jboss.org/browse/ISPN-10284) for it, but then I realized processor path is a bit wonky in Maven so I closed it.